lib/plugins/enhancedPdf417Plugin.js

```diff
@@ -198,22 +265,22 @@ export const enhancedPdf417Plugin = {
   format: 'pdf417_enhanced',
 
   /**
-   * Scan source for PDF417 codes using enhanced algorithms.
-   *
-   * @param {HTMLImageElement|HTMLVideoElement|HTMLCanvasElement|
-   *  ImageData|File} source - Source to scan.
-   * @param {object} options - Plugin-specific options.
-   * @param {AbortSignal} options.signal - Abort signal.
-   * @param {string} options.license - Dynamsoft license key.
-   * @param {boolean} options.useDynamsoft - Use Dynamsoft engine.
-   * @param {boolean} options.parseDL - Parse driver license info.
-   * @param {number} options.deblurLevel - Deblur level 1-9.
-   * @param {number} options.regionScale - Region scale 0-1.
-   * @param {boolean} options.useRegion - Enable region-based scanning.
-   * @param {boolean} options.fallbackToBarcodeDetector - Fallback enabled.
-   *
-   * @returns {Promise<object[]>} Array of detected PDF417 codes.
-   */
+  * Scan source for PDF417 codes using enhanced algorithms.
+  *
+  * @param {HTMLImageElement|HTMLVideoElement|HTMLCanvasElement|
+  *  ImageData|File} source - Source to scan.
+  * @param {object} options - Plugin-specific options.
+  * @param {AbortSignal} options.signal - Abort signal.
+  * @param {string} options.license - Dynamsoft license key.
+  * @param {boolean} options.useDynamsoft - Use Dynamsoft engine.
```

Comment:
Use markup for all the optional params here:
```suggestion
  * @param {boolean} [options.useDynamsoft] - Use Dynamsoft engine.
```

Reply:
@davidlehn 

Noted. I will use `[]` if it is an optional parameter whenever it is applicable.